.changeset/wicked-worms-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"ci-prettier": patch
```

Comment:
should this be `minor`?

Reply:
Thanks, updated in 8f164b2f1934e17f28d953350aaea6d6d2efcb2a